backend/benefit/applications/tests/conftest.py

```diff
@@ -1293,5 +1280,5 @@ def non_expired_token():
         access_token="access_token",
         refresh_token="refresh_token",
         expires_in=30000,
-        created_at=datetime.now(timezone.utc),
+        created_at=datetime.now(tz.utc),
```

Comment:
`django.utils.timezone.now` should work as well (since `USE_TZ` is `True`);

```suggestion
        created_at=timezone.now(),
```